types/react-overlays/lib/DropdownToggle.d.ts

```diff
@@ -10,7 +10,7 @@ declare namespace DropdownToggle {
         show: boolean;
         toggle(show: boolean): void;
         props: {
-            ref?(element: HTMLElement): void;
+            ref: (popperNode: HTMLElement|null) => void;
```

Comment:
Make sure to run the linter if you haven't already. I'm not sure if the lack of spaces around the pipe is an issue or not.